packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -300,21 +300,21 @@ void main() {
             child: PopupMenuButton<void>(
               child: const Text('Actions'),
               itemBuilder: (BuildContext context) => <PopupMenuItem<void>>[
-                  PopupMenuItem<void>(
-                      child: const Text('First option'),
-                      onTap: () {
-                          menuItemTapCounters[0]++;
-                      },
-                  ),
-                  PopupMenuItem<void>(
-                      child: const Text('Second option'),
-                      onTap: () {
-                          menuItemTapCounters[1]++;
-                      },
-                  ),
-                  const PopupMenuItem<void>(
-                      child: Text('Option without onTap'),
-                  ),
+                PopupMenuItem<void>(
+                  child: const Text('First option'),
+                  onTap: () {
+                    menuItemTapCounters[0] += 1;
```

Comment:
Is this a personal preference for += 1 instead of ++ or is this something spec'd in the style guide?

Reply:
Ian asked for += 1 instead of ++ in a review ages ago, so I've developed that habit.

Reply:
see the style guide :-)

Reply:
FTR: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-over

Reply:
Yup thx. Sorry for not just looking it up in the first place 😮. Also not sure why it didn't sink in for me given that I have read through that twice before. Must be too many style guides in my past starting to blend together.